### DISCUSSION TITLE
[Bug]: Move the mouse out of the window or touch the bottom menu

### DISCUSSION THREAD

**Discussion by Xing-YaoLei** [PlayCover/PlayCover#728] [General]
### Describe the bug

I use a Macbook Pro with M1Pro
Game：Ginshin
issue：During the game through the keyboard mapping to complete the use of keys，However, when moving the Angle of view, the mouse will slide, but the program does not seem to block the operation of the mouse outside the window. As a result, I often encounter that the mouse moves outside the window, creates a folder or mistakenly clicks on a menu below the Mac and loses control.

### Steps to reproduce

1、Using keyboard mapping
2、Create a mouse map to move the perspective around
3、Slide the mouse, mouse out the window

### Expected behaviour

I think the program should either block the mouse or hide it when the option button is not pressed

### What version of PlayCover are you using?

Nightly/beta

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by Xing-YaoLei** [comment]
I seem to have found a temporary solution that simply hides the mouse every time you use it by pressing the option key. The need to re-use the command+k key to open and close the mouse seems to solve the problem of moving the mouse out of the window

**Comment by Depal1** [comment]
You should enable `Mouse Mapping` https://docs.playcover.io/settings/keymapping

**Reply by idler8** [reply]
It doesn't work

**Comment by idler8** [comment]
It's like the mouse is unlocked and just hidden
